rt/ws/security/src/main/java/org/apache/cxf/ws/security/trust/AbstractSTSClient.java

```diff
@@ -1453,7 +1453,7 @@ protected void addClaims(XMLStreamWriter writer) throws Exception {
         if (claimsToSerialize instanceof Element) {
             StaxUtils.copy((Element)claimsToSerialize, writer);
         } else if (claimsToSerialize instanceof ClaimCollection) {
-            ClaimCollection claimCollection = (ClaimCollection)claims;
+            ClaimCollection claimCollection = (ClaimCollection)claimsToSerialize;
```

Comment:
Thanks for the fix @lukepasek , seems like we have a gap in tests, could you please add at least one to capture that? Thank you!

Reply:
I'll add the test to the STS systests